src/v2/commands/npm/index.ts

```diff
@@ -204,8 +202,10 @@ const createNPMEmbed = (
     })
     .setURL(url)
     .setFooter(
-      `requested by ${user.username}#${user.discriminator}`,
-      user.avatarURL({ size: 64, format: 'webp' })
+      {
+        text: `requested by ${user.username}#${user.discriminator}`,
```

Comment:
question: should we still support discriminators, (discord removed them no?)

Reply:
Removed.